resolwe/elastic/viewsets.py

```diff
@@ -128,7 +128,7 @@ def filter_permissions(self, search):
 class PaginationMixin(object):
     """Mixin for making paginated response in case pagination parameters are provided."""
 
-    def paginate_response(self, queryset):
+    def paginate_response(self, queryset, **serializers_kwargs):
```

Comment:
Use `serializers_kwargs` instead of `**serializers_kwargs`, otherwise it will be hard to pass attributes for other functions if needed.